packages/block/src/types.ts

```diff
@@ -93,7 +93,7 @@ export interface BlockData {
    * Header data for the block
    */
   header?: HeaderData
-  transactions?: Array<TxData>
+  transactions?: Array<TxData | AccessListEIP2930TxData>
```

Comment:
This may look like a breaking change, but it's not, as an `AccessListEIP2930TxData` is a `TxData`, right?

Reply:
yes it simply extends `TxData` with 3 new optional fields: chainId, accessList, type.